src/graphql/resolvers/mempool-resolver.ts

```diff
@@ -88,11 +95,31 @@ export class MempoolResolver {
 
   @Mutation(() => String)
   async checkTransaction(@Arg("signedTransaction") signedTransaction: SignedTransactionInput) {
-    return "2ab9da11fc216660e974842cc3b7705e62ebb9e0bf5ff78e53f9cd40abadd117";
+    try {
+      const response = await this.nodeservice.checkTransaction(signedTransaction);
+      return response.data;
+    } catch (e) {
+      if(e instanceof AxiosError){
+        const error = e.response?.data;
+        if(error.error === 400)
+          throw new GraphQLError(error.detail);
+      }
+      throw new GraphQLError("Unknown error");
+    }
   }
 
   @Mutation(() => String)
   async submitTransaction(@Arg("signedTransaction") signedTransaction: SignedTransactionInput) {
-    return "2ab9da11fc216660e974842cc3b7705e62ebb9e0bf5ff78e53f9cd40abadd117";
+    try {
+      const response = await this.nodeservice.submitTransaction(signedTransaction);
+      return response.data;
+    } catch (e) {
+      if(e instanceof AxiosError){
+        const error = e.response?.data;
+        if(error.error === 400)
+          throw new GraphQLError(error.detail);
+      }
+      throw new GraphQLError("Unknown error");
```

Comment:
Let's help future troubleshooting by logging this error using console.error(e) before throwing the exception.